dsl/build.gradle

```diff
@@ -83,6 +84,7 @@ kotlin {
 tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
     kotlinOptions {
         freeCompilerArgs += ["-Xuse-experimental=kotlin.Experimental"]
+        jvmTarget = "1.8"
```

Comment:
Needed because flMadStateMachine is built with `1.8`.

Reply:
👍